### ISSUE TITLE
Data Page - dataset view suggestions

### ISSUE THREAD

**Issue by rondlg** [gbif/hosted-portals#19]
Hi,

On this page: https://hp-north-america.gbif-staging.org/data?view=dataset

1) Is it possible to turn off/change/remove the green "How many records" bar for the entries on this page? Given that eBird will always be the biggest player it seems a less than useful visual especially as the number of records is also displayed. 

2) Can we override the ordering? Perhaps to alphabetical.

3) Or even better just be like the existing GBIF dataset view https://www.gbif.org/dataset/search?q=   

Thanks

Sharon

**Comment by MortenHofft** [comment]
You are hitting a sore spot. I'm really in doubt what to do with datasets and publishers. The current version is really mostly there to start the discussion about what is needed and to have _something_

**"eBird will always be the biggest player"**
It is really mostly intended as a list of datasets contributing occurrences to a filtered search. E.g. what are the datasets holding information about [Basidiomycota in Alaska](https://hp-north-america.gbif-staging.org/data?filter=eyJtdXN0Ijp7ImdhZG1HaWQiOlsiVVNBLjJfMSJdLCJ0YXhvbktleSI6WzM0XX19&view=dataset)
That is also serves as a list of all datasets when there is no filter is more of a nice coincidence.

**Green bar**
We default to logarithmic on [gbif.org](https://www.gbif.org/occurrence/charts?country=US&occurrence_status=present&d=datasetKey), but like you I'm tempted to remove it altogether. Perhaps do it as a table instead.

**Override ordering**
Unlike gbif.org dataset search, then this subset is defined by the occurrences. Meaning it will only show datasets with occurrences. And we can only really access these through a facet, meaning it will be ordered by counts.

It is essentially doing this https://api.gbif.org/v1/occurrence/search?country=US&limit=0&facet=datasetKey

**Pagination**
You didn't mention it, but this page need pagination as well

**Search**
You can filter using the occurrence filters, but that doesn't include all the filters from  https://www.gbif.org/dataset/search?q= (e.g. not dataset type)

**Suggestion**
Would it make sense to have a occurrence metrics tab + a more dedicated dataset search (not a tab on occurrence search)?

_Metrics tab_
In many ways I consider the current tab a temporary measure until we have a metrics tab with arbitrary breakdowns similar to [gbif.org/charts](https://www.gbif.org/occurrence/charts?country=US&occurrence_status=present&d=datasetKey)

_Dataset search_
I think we should add datasets as a content type of its own, instead of a tab on occurrence search. And this would be more similar to [gbif.org/dataset/search](https://www.gbif.org/dataset/search?q=) (which doesn't order alphabetically either). 

Again the challenge is how to include datasets without occurrences and secondly how to sort them (if size isn't desired). This is a challenge because you cannot filter datasets in the API to match your data scope.

**Comment by albenson-usgs** [comment]
Yes pagination for the datasets tab is definitely needed. Does it need its own ticket Morten?

**Comment by MortenHofft** [comment]
I'm closing this as we have long had the option for a dataset search instead. The dataset tab is a patch for those cases where there is no meaningful dataset search scope. 

This is better done by the existing https://hp-north-america.gbif-staging.org/dataset/search
The dataset tab is about telling users who contributed data to this specific filtered subset. Which makes the red bar relevant.

We also have dahsboards for these breakdowns so in most cases i suspect the dataset tab is less useful, albeit an easy option
